set.mm

```diff
@@ -48685,16 +48715,11 @@ although the definition does not require it (see ~ dfid2 for a case
     $( An equality inference for the maps-to notation.  (Contributed by NM,
        24-Aug-2011.)  (Revised by Mario Carneiro, 16-Dec-2013.)  Drop ~ ax-10
        while shortening its proof.  (Revised by Steven Nguyen and Gino Giotto,
-       1-Dec-2023.) $)
+       1-Dec-2023.)  Drop ~ ax-12 (recovering the OLD version).  (Revised by
```

Comment:
Probaly no need for the parenthetical comment about the OLD version. Actually, you may even merge the latest two revision comments into a single "Remove dependency on ~ ax-10 , ~ ax-12 ."